### PR TITLE
LVM-489: Added header, buttons, for assessment index view

### PR DESCRIPTION
**Pull request by xingyif**
https://cs5500-jira.ccs.neu.edu/browse/LVM-489

screen shot 2017-04-13 at 20 21 10

### CODE REVIEW

File: app/views/affiliates/index.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   </div>
 </div>
 
-<table id="affiliateTable" class"display hover order-column compact" width="100%">
```
Comment (zack) [inline app/views/affiliates/index.html.erb:7]: hahahahahahahaha

File: app/views/assessments/index.html.erb [hunk 2]
```diff
@@ -1,36 +1,74 @@
...
-<table>
+<table id="assessmentTable" class="display hover order-column compact" width="100%">
   <thead>
     <tr>
-      <th>Category</th>
-      <th>Name</th>
-      <th>Level</th>
-      <th>Assessment Type</th>
-      <th>Score</th>
-      <th>Assessment Date</th>
```
Comment (zack) [inline app/views/assessments/index.html.erb:11]: YESSSS spaces

File: app/views/assessments/index.html.erb [hunk 3]
```diff
@@ -1,36 +1,74 @@
...
+        <td class="text-right">
+          <%= link_to edit_assessment_path(assessment)do %>
+              <button class="btn btn-primary btn-sm">
+                <span class="glyphicon glyphicon-pencil"></span>
+              </button>
+          <% end %>
+          <%= link_to assessment_path(assessment), method: :delete, data: { confirm: 'Are you sure you want to delete this
+          assessment?'
+          } do %>
+              <button class="btn btn-danger btn-sm"><span class="glyphicon glyphicon-remove"></span></button>
```
Comment (zack) [inline app/views/assessments/index.html.erb:37]: As much as I HATE the words, Roberta said that having the words "edit" and "delete" on these buttons would be good for some of her less technical users.

**Review by zack — APPROVED** [review]
Please make tickets (or just one ticket) to bring all index views into the same style.

File: app/views/assessments/index.html.erb (hunk 3 above)
Comment (xingyif) [reply app/views/assessments/index.html.erb:37]: what!? icons are much more informative! they literally don't have to READ!
... oh well. if that's what she wants

File: app/views/affiliates/index.html.erb (hunk 1 above)
Comment (xingyif) [reply app/views/affiliates/index.html.erb:7]: what's so funny :)

Comment (xingyif) [reply app/views/affiliates/index.html.erb:7]: oh, haha, your fault.

**Comment by xingyif** [comment]
@zack you got it! will do prob. tmr/sat

File: app/views/assessments/index.html.erb (hunk 3 above)
Comment (zack) [reply app/views/assessments/index.html.erb:37]: I know. I agree. I COMPLETELY agree.
If we ... if we really want we can remove all of the words for tomorrow and see how people respond? You would think that pencil, magnifying glass, and a X should be pretty universal.
I'll ask some UX people at work tomorrow.

**Pull request merged**